pkg/runtime/config.go

```diff
@@ -40,4 +41,10 @@ func (c *Config) BindFlags() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.",
 	)
+	flag.BoolVar(
+		&c.EnableDevModeLogging, "enable-dev-mode-logging",
```

Comment:
I'd prefer naming the option `enable-development-logging`. The shortened term "dev" is easily confused with "device" :)

Reply:
done,